src/main/java/g0301_0400/s0349_intersection_of_two_arrays/Solution.java

```diff
@@ -0,0 +1,23 @@
+package g0301_0400.s0349_intersection_of_two_arrays;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// #Easy #Array #Hash_Table #Sorting #Binary_Search #Two_Pointers
+
+public class Solution {
+    public int[] intersection(int[] nums1, int[] nums2) {
+        List<Integer> intersectionList = new ArrayList<>();
+
+        for (int j : nums1) {
+            for (int k : nums2) {
+                if (j == k && !intersectionList.contains(j)) {
+                    intersectionList.add(j);
```

Comment:
You may use solution filter.

![image](https://user-images.githubusercontent.com/348496/145668435-b7b42a5d-8730-4d32-a735-66645edb068e.png)

Reply:
Ok, I will update